### PR TITLE
Tests: Added test_determine_symmetry to common tests

### PR DESCRIPTION
**Pull request by alongd**
The symmetry script (from the RMG qm module) was not tested in ARC. Added some simple tests.

### CODE REVIEW

File: arc/commonTest.py [hunk 1]
```diff
@@ -827,6 +827,31 @@ def test_get_bonds_from_dmat(self):
         self.assertTrue(common.check_that_all_entries_are_in_list(
             bonds, [(0, 1), (1, 2), (0, 3), (0, 4), (1, 6), (2, 7), (2, 8), (2, 9), (5, 10)]))  # No (1, 5) bond.
 
+    def test_determine_symmetry(self):
+        """Test the determine_symmetry() function"""
```
Comment (Laxzal) [inline arc/commonTest.py:831]: changes look good. the only thing I would like added is just a bit more explanation of the test.
inlcuding the phrase
""" Test the determine symmetry function"""
Also include about which the 3 tests in this function are doing and what you are expecting. I just think adding a bit more description will help when this test is run again in the future by someone else.

**Review by Laxzal — CHANGES_REQUESTED** [review]

**Comment by alongd** [comment]
Thanks, @Laxzal. Added and squashed.

**Comment by alongd** [comment]
@Laxzal, I think we can merge this in once tests pass

**Review by kfir4444 — CHANGES_REQUESTED** [review]
Thanks @alongd!
I think the tests misses one case, where there are both symmetry and chirality. Could you add it?
Example: `CC=C=CC`

**Comment by alongd** [comment]
Thanks for the suggestion, @kfir4444, added

**Review by kfir4444 — APPROVED** [review]
Thanks for addressing the comments @alongd,
approved and merged

**Pull request merged**